lib/handler/redirect-handler.js

```diff
@@ -239,3 +240,4 @@ function cleanRequestHeaders (headers, removeContent, unknownOrigin) {
 }
 
 module.exports = RedirectHandler
+module.exports.cleanRequestHeaders = cleanRequestHeaders
```

Comment:
Let's not do that if we are just exporting it to testing it out.

You can attempt to test the same on the [`redirect-request`](https://github.com/nodejs/undici/blob/main/test/redirect-request.js) test suite where you submit a `Headers` instance instead of the array/object way.

Reply:
Added tests in appropriate file